README.md

```diff
@@ -0,0 +1,77 @@
+# iOS Enhanced media player
+
+Repo with iOS media player package for other projects. Implemented in SwiftUI using native API: AVFoundation.
+
+## Table of Contents
+
+- [Motivation](#motivation)
+- [Minimum deployment target](#minimum-deployment-target)
+- [Integration](#integration)
+- [Features](#features)
+- [Visual references](#visual-references)
+
+## Motivation
+
+Internal project focused on learning, training, and practicing media player development. Our goal is to build a feature-rich media player, akin to those found in commercial platforms like YouTube, using SwiftUI and other contemporary development practices. This project serves as a valuable resource for future reference, providing hands-on experience in media player creation for iOS platforms.
```

Comment:
@KarenBaliero when submitting PRs for markdown break the lines on a certain number of characters. This makes it easier to add comments on specific lines instead of whole paragraphs, and it's also better for editors that don't auto-wrap the lines. You can probably add a `VSCode` (or `nvim`) extension to help with auto-formatting

Reply:
Thanks for suggestion, I used this tool: VS Marketplace Link: https://marketplace.visualstudio.com/items?itemName=a-bentofreire.vsctoix

Reply:
Changed for this one: https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint but had to create a file to enable the rule